examples/digital-asset-manager/netlify/functions/redirect.js

```diff
@@ -0,0 +1,76 @@
+import fetch from "node-fetch";
+import { serialize } from "cookie";
+
+exports.handler = (event) => {
+  const code = event.queryStringParameters.code;
+
+  const encodedData = Buffer.from(
+    process.env.CLIENT_ID + ":" + process.env.CLIENT_SECRET
+  ).toString("base64");
+
+  const details = {
+    code: code,
+    grant_type: "authorization_code",
+    scope: "offline asset:read collection:read",
+    client_id: process.env.CLIENT_ID,
+    redirect_uri: process.env.REDIRECT_URI,
+  };
+
+  var formBody = [];
+
+  for (var property in details) {
+    var encodedKey = encodeURIComponent(property);
+    var encodedValue = encodeURIComponent(details[property]);
+    formBody.push(encodedKey + "=" + encodedValue);
+  }
+  formBody = formBody.join("&");
+
+  return fetch(`${process.env.BASE_URL}/v6/authentication/oauth2/token/`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/x-www-form-urlencoded",
+      Authorization: `Basic ${encodedData})`,
+    },
+    body: formBody,
+  })
+    .then((response) => {
+      return response.json();
+    })
+    .then((result) => {
+      // Set response in cookies
+      const hour = 3600;
+
+      const access_token = serialize(
+        "bynder_access_token",
+        result.access_token,
+        {
+          secure: true,
+          httpOnly: true,
+          sameSite: "None",
+          path: "/",
+          maxAge: hour,
+        }
+      );
+
+      const refresh_token = serialize(
+        "bynder_refresh_token",
+        result.refresh_token,
+        {
+          secure: true,
+          httpOnly: true,
+          sameSite: "None",
+          path: "/",
+        }
+      );
+
+      return {
+        statusCode: 301,
+        Location: process.env.PRODUCTION_URL,
+        multiValueHeaders: {
+          "Set-Cookie": [refresh_token, access_token],
+        },
+        body: "<script>window.close();</script>",
+      };
+    })
+    .catch((error) => console.error(error));
```

Comment:
There isn't because this function isn't called from the client, it's called from the redirect from Bynder. This is a bit related to the question you've asked below about the 1 second interval. 

Maybe, if there is an error, I could set a cookie called error (or something similar), and check for this in the `authorize` endpoint (And also show a visual error as needed)?